automation/test.sh

```diff
@@ -201,7 +211,7 @@ set -e
 make bazel-build-images
 
 trap '{ echo "Dump kubevirt state:"; make dump; }' ERR
-make cluster-up
+make cluster-up || collect_debug_logs
```

Comment:
Why this has been moved here from the `trap`?

Reply:
also take into account that even with trap, here there is trap - ERR on line 215
best to trap globally i think, in any case of error ?

Reply:
line 214 suppress line 213 trap

Reply:
Or is right, on failure of `make cluster-up`, the trap will not run `make dump` (because of the OR chain).
See suggestion above.

Reply:
Good catch. I've updated the PR.